tasks/sep/fp-recovery/005-set-game-implementation/justfile

```diff
@@ -0,0 +1,91 @@
+set positional-arguments
+
+# default recipe to display help information
+default:
+  @just --list
+
+clean:
+  rm -rf out
+
+prep l1='' l2='':
+  #!/usr/bin/env bash
+  set -euo pipefail
+  mkdir out
+  sed -e 's/<L2_CHAIN>/{{l2}}/g' -e 's/<L1_CHAIN>/{{l1}}/g' templates/README.md > out/README.md
```

Comment:
Will this work cross platform? I know sed has some linux vs macOS differences but offhand I can't remember what they were

Reply:
Yeah good call.  I tested it on linux this one works and is pretty basic sed syntax I've been using for years.  The ones below did need adjusting because the -I option wasn't on linux (just -i).